doc/arkode/guide/source/Usage/User_callable.rst

```diff
@@ -47,8 +47,12 @@ E. functions that apply for time-stepping modules that support relaxation Runge-
 
 In the function descriptions below, we identify those that have any of the restrictions B-E above.
 Then in the introduction for each of the stepper-specific documentation sections
-(:numref:`ARKODE.Usage.ARKStep.UserCallable`, :numref:`ARKODE.Usage.ERKStep.UserCallable`,
-:numref:`ARKODE.Usage.MRIStep.UserCallable`, and :numref:`ARKODE.Usage.SPRKStep.UserCallable`)
+(:numref:`ARKODE.Usage.ARKStep.UserCallable`,
+:numref:`ARKODE.Usage.ERKStep.UserCallable`,
+:numref:`ARKODE.Usage.SplittingStep.UserCallable`,
+:numref:`ARKODE.Usage.ForcingStep.UserCallable`,
+:numref:`ARKODE.Usage.MRIStep.UserCallable`,
+and :numref:`ARKODE.Usage.SPRKStep.UserCallable`)
```

Comment:
Is there some logic behind inserting the new steppers in the middle of this list?  In the absence of some clear justification, I think that this should retain the chronological ordering, with the new steppers put at the end of the list.